ocfweb/stats/urls.py

```diff
@@ -27,4 +27,5 @@
     ),
     url(r'^printing/semester-histogram/graph$', semester_histogram, name='semester_histogram'),
     url(r'^printing/pages-printed$', pages_printed, name='pages_printed'),
+    url(r'^daily-job/graph$', daily_jobs_image, name='daily_job_image'),
```

Comment:
Never thought of that but it does sound more reasonable.
P.S: I am not sure how the whole url works so I just "copy"  the code from "daily_graph_image"